elementary/utils/time.py

```diff
@@ -39,3 +39,10 @@ def format_milliseconds(duration: int) -> str:
     )
 
     return f"{hours}h:{minutes}m:{seconds}s:{remaining_milliseconds}ms"
+
+
+def try_parse_time(isoformat_datetime: str) -> str:
```

Comment:
WDYT about renaming to convert_isoformat_to_custom_format and receive the format as default DATETIME_FORMAT?